src/hardware/addon-protocols/rs232-usb.md

```diff
@@ -583,3 +590,119 @@ This is an example of binary data packets for image data transferred using the M
 | Total Number of Payload Bytes Received | x | 12 |
 | Checksum | 2 | 12+x |
 | ETX (0x03) | 1 | 14+x |
+
+### Appendix D: Using protobuf Messages to Communicate by using 0x8C, 0x26
```

Comment:
In general, we are using protobuf itself in documentation, why not just copy proper .proto file itself. It is really confusing right now as the documentation is using mix of non-proto format comments and protobuf message formats.

Instead, comments can be done using // in the proto file itself. Comment should not describe what is defined but should say how and where and when it is used.